src/supplemental/nanolib/conf_ver2.c

```diff
@@ -628,6 +628,13 @@ conf_auth_parse_ver2(conf *config, cJSON *jso)
 			    auth->usernames, nng_strdup(ele->string));
 			cvector_push_back(
 			    auth->passwords, nng_strdup(ele->valuestring));
+		} else if(cJSON_IsNumber(ele)) {
+			cvector_push_back(
+			    auth->usernames, nng_strdup(ele->string));
+			// max len of pw is 20.
+			char strpw[20];
+			sprintf(strpw, "%d", ele->valueint);
+			cvector_push_back(auth->passwords, nng_strdup(strpw));
```

Comment:
I don't think the password can be a number

Reply:
just revert it, gonna update docs and tell users to add `""` .